tests/conductivity_test.py

```diff
@@ -0,0 +1,232 @@
+
+        """
+        # parallel
+        config.numcores = -1
+
+        # set up the atom and model
+        F_at = Atom("F", 0.4, radius=5.0)
+        model = models.ISModel(F_at, unbound="quantum", bc="dirichlet")
+
+        # run the SCF calculation
+        output = model.CalcEnergy(
+            4, 4, scf_params={"mixfrac": 0.3, "maxscf": 6}, grid_params={"ngrid": 1200}
+        )
+
+        return output
+
+    @staticmethod
+    def _run_cond_tot(input_SCF, which, val_orb):
+        """
+        Compute the number of conducting electrons via the cond_tot function.
+
+        Parameters
+        ----------
+        input_SCF : dict of objects
+            the SCF input
+        which : the componenet for which the number of electrons is computed.
+        val_orb : tuple of ints
+            defines the valance orbital
+
+        Returns
+        -------
+        N : float
+            number of conductiong electrons in the component.
+
+        """
+        cond = conductivity.KuboGreenwood(input_SCF["orbitals"], valence_orbs=[val_orb])
+
+        N = cond.cond_tot(component=which)[1]
+        return N
+
+    @staticmethod
+    def _run_prop(input_SCF, which):
+        """
+        Return properties from the Kubo-Greenwood class.
+
+        Parameters
+        ----------
+        input_SCF : dict of objects
+            the SCF input
+        which : the property to be calculated
+
+        Returns
+        -------
+        prop: float
+            The property's value
+        """
+        cond = conductivity.KuboGreenwood(input_SCF["orbitals"], valence_orbs=[(2, 0)])
+        if which == "sig_vv":
+            prop = cond.sig_vv
+        elif which == "sig_cv":
+            prop = cond.sig_cv
+        elif which == "N_tot":
+            prop = cond.N_tot
+        else:
+            prop = cond.N_free
+        return prop
+
+    @staticmethod
+    def _run_int_calc(order):
+        """
+        Calculate a spherical harmonic integral.
+
+        Parameters
+        ----------
+        order : int
+            the order of the P integral to be calculated.
+
+        Returns
+        -------
+        integ: float
+            the resulting integral.
+
+        """
+        spherical_har = conductivity.SphHamInts()
+        integ = spherical_har.P_int(order, 1, 2, 1)
+        return integ
+
+    @staticmethod
+    def _run_sum_rule(input_SCF):
+        """
+        Check the sum rule of an unconveged set of orbitals.
+
+        Parameters
+        ----------
+        input_SCF : dict of objects
+            the SCF input
+
+        Returns
+        -------
+        sum_rule : float
+            the result of trying to apply the sum rule to an
+            arbitrary inconverged orbital.
+
+        """
+        cond = conductivity.KuboGreenwood(input_SCF["orbitals"])
+        sum_rule = cond.check_sum_rule(2, 2, 0)[0]
+        return sum_rule
+
+
+# if __name__ == "__main__":
```

Comment:
Please uncomment this section and add all the necessary statements that would reproduce the expected numbers at the start of the file. This makes debugging much easier! 

(`if __name__ == "__main__":` means that these commands will only be run if the script is called directly, so it's not necessary to comment them out)